brave-lists/clean-urls.json

```diff
@@ -73,6 +73,37 @@
             "ref_"
         ]
     },
+    {
+        "include": [
+            "*://*.ebay.com/itm/*",
+            "*://www.ebay.at/itm/*",
+            "*://www.ebay.ca/itm/*",
+            "*://www.ebay.co.uk/itm/*",
+            "*://www.ebay.com.au/itm/*",
+            "*://www.ebay.com.sg/itm/*",
+            "*://www.ebay.de/itm/*",
+            "*://www.ebay.es/itm/*",
+            "*://www.ebay.ie/itm/*",
+            "*://www.ebay.fr/itm/*",
+            "*://www.ebay.it/itm/*",
+            "*://www.ebay.ch/itm/*",
+            "*://www.ebay.nl/itm/*",
+            "*://www.ebay.pl/itm/*"
+        ],
+        "exclude": [
+
+        ],
+        "params": [
```

Comment:
Interesting, none of these are on the [AdGuard list](https://github.com/AdguardTeam/AdguardFilters/blob/c1f54ae78eb03858d81a8439999980dd3a57e833/TrackParamFilter/sections/specific.txt#L1351-L1356).

Reply:
Can ask Adguard to review this also.

Reply:
`mkcid`, `mkevt` aff params are ok to block.
Can't find where the rest are used. Could you please describe where can I find them?

Reply:
> Can't find where the rest are used. Could you please describe where can I find them?

We've not found documentation on what these are used for, but you can see an example in the issue description.

Reply:
Ok, added also `ss*` parameters.
Please mention us if users will report about breakage:)